### PR TITLE
Update README and startup script

### PR DESCRIPTION
**Pull request by 1pretz1**
Updates README in favour of GOVUK Docker and limits startup script to only operate in `live` mode.

---

Trello:
https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
 # Info Frontend
 
-This application exposes the user needs and metrics about content on GOV.UK.
+This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching content details (including user needs) from [Content Store](https://github.com/alphagov/content-store).
 
-It fetches content details (including user needs) from the [`content-store`](https://github.com/alphagov/content-store).
+Information pages have a prefix of `/info`, for example: https://www.gov.uk/info/benefits-calculators and https://www.gov.uk/info/employers-sick-pay
```
Comment (benthorner) [inline README.md:5]: Minor: we could shorten these to `/benefits-calculators` and `employers-sick-pay` links.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,23 @@
 # Info Frontend
 
-This application exposes the user needs and metrics about content on GOV.UK.
+This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching content details (including user needs) from [Content Store](https://github.com/alphagov/content-store).
```
Comment (benthorner) [inline README.md:3]:
```suggestion
This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching from [Content Store](https://github.com/alphagov/content-store).
```

Minor: I don't think we need to harp on about the needs ;-).

File: README.md [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
+
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
+
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+
+### Running the test suite
+
+```
+bundle exec rspec
```
Comment (benthorner) [inline README.md:18]:
```suggestion
bundle exec rake
```

**Review by benthorner — APPROVED** [review]
Thanks Pete! Just a few comments, but nothing major.

On a side note: I'm wondering if we can delete the `Dockerfile` this repo has? I don't think it's used by the E2E tests, and appears to have been added [speculatively](https://github.com/alphagov/info-frontend/commit/45b2da2d786a9ada260aac545b543fad0c658f14#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557).

File: README.md (hunk 2 above)
Comment (1pretz1) [reply README.md]: lol but it's allllllllllways about user needs 😆 

Have removed some fluff from the sentence. 👍

File: README.md (hunk 1 above)
Comment (1pretz1) [reply README.md:5]: Hmm, I like them as full links as I think it's clearer and the person reading doesn't have to click through to the page to get the gist of the comment.

**Comment by 1pretz1** [comment]
Thanks for the review @benthorner! Sorry for the delayed comments, I was on AL Friday. I've also deleted the Dockerfile as recommended as I too couldn't find where it was used.

**Pull request merged**